src/ucp/wireup/address.c

```diff
@@ -1057,30 +1070,51 @@ static void ucp_address_pack_header_flags(uint8_t *address_header,
                                           uint8_t flags)
 {
     if (addr_version == UCP_OBJECT_VERSION_V1) {
-        *address_header |= (flags << UCP_ADDRESS_HEADER_FLAGS_SHIFT_V1);
+        *address_header |= (flags << UCP_ADDRESS_HEADER_SHIFT);
     } else {
         address_header += 1;
         *address_header = flags;
     }
 }
 
+static void *ucp_address_pack_header(uint8_t *ptr,
```

Comment:
adding version to ucp address could have been a separate pr, since this one is already quite big

Reply:
ok